diff -urN ndesk-dbus-0.6.0/src/Makefile.am ndesk-dbus-0.6.0.new/src/Makefile.am
--- ndesk-dbus-0.6.0/src/Makefile.am	2007-10-13 21:43:51.000000000 +0200
+++ ndesk-dbus-0.6.0.new/src/Makefile.am	2007-12-13 19:47:21.480086808 +0100
@@ -40,14 +40,14 @@
 
 install-data-local:
 	@if test -n '$(TARGET)'; then                       \
-          echo "$(GACUTIL) /i $(TARGET) /f /gacdir $(DESTDIR)$(libdir)";    \
-          $(GACUTIL) /i $(TARGET) /package $(PACKAGE)-$(API_VERSION) /f /gacdir $(libdir) /root $(DESTDIR)$(libdir) || exit 1; \
+          echo "$(GACUTIL) /i $(TARGET) /f /gacdir $(DESTDIR)$(prefix)/lib";    \
+          $(GACUTIL) /i $(TARGET) /package $(PACKAGE)-$(API_VERSION) /f /gacdir $(libdir) /root $(DESTDIR)$(prefix)/lib || exit 1; \
         fi
 
 uninstall-local:
 	@if test -n '$(TARGET)'; then                       \
-          echo "$(GACUTIL) /u $(ASSEMBLY) /gacdir $(DESTDIR)$(libdir)";  \
-          $(GACUTIL) /u $(ASSEMBLY) /package $(PACKAGE)-$(API_VERSION) /gacdir $(libdir) /root $(DESTDIR)$(libdir) || exit 1;   \
+          echo "$(GACUTIL) /u $(ASSEMBLY) /gacdir $(DESTDIR)$(prefix)/lib";  \
+          $(GACUTIL) /u $(ASSEMBLY) /package $(PACKAGE)-$(API_VERSION) /gacdir $(libdir) /root $(DESTDIR)$(prefix)/lib || exit 1;   \
         fi
 
 EXTRA_DIST = \
